contrib/gocql/gocql/gocql.go

```diff
@@ -242,7 +242,8 @@ func (tq *Query) Iter() *Iter {
 	if tIter.Host() != nil {
 		tIter.span.SetTag(ext.TargetHost, tIter.Iter.Host().HostID())
 		tIter.span.SetTag(ext.TargetPort, strconv.Itoa(tIter.Iter.Host().Port()))
-		tIter.span.SetTag(ext.CassandraCluster, tIter.Iter.Host().DataCenter())
```

Comment:
We should be careful before merging this change, as it can be a breaking change for anyone relying on the value of this tag.

Reply:
Agree, we should be careful, but for now there's nonetheless a problem with the data being reported, `datacenter` is not `cluster`, and this is misleading when looking at span metadata.